tasks/configuration/increaseCap.js

```diff
@@ -0,0 +1,29 @@
+task('increaseCap', 'Increase an option or pool cap')
+  .addParam('contract', 'An address of a target address to increase cap (either option or pool')
+  .addParam('value', 'new cap number')
```

Comment:
If the `max` flag ignores the `value` then it should be optional
```suggestion
  .addOptionalParam('value', 'new cap number')
```